add VSCode remote container env (#1)
spadgett/console-plugin-template: README.md

## Local development

> ### Option 1: VSCode Remote Container

Reviewer: I would call this option 2 since it relies on a VSCode plugin (and requires VSCode).

Make sure [Remote Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension is installed. 
This method uses docker compose with one container being the console and the bridge,
and the second container being the plugin with VSCode. After initial build, the cached containers will help you start developing in seconds.

> Make sure you have 8GB of RAM or more dedicated to docker.

Reviewer: You'll be able to remove this line if we use the origin-console image.

> 1. create `dev.env` file inside `.devcontainer` folder with the following values:

Reviewer: ```suggestion
1. create `dev.env` file inside `.devcontainer` folder with the correct values for your cluster:
```

```bash

> OC_PLUGIN_NAME=console-plugin-template
> OC_URL=
> OC_USER=
> OC_PASS=

Reviewer: ```suggestion
OC_PLUGIN_NAME=my-plugin
OC_URL=https://api.example.com:6443
OC_USER=kubeadmin
OC_PASS=<password>
```

```

> 2. Make sure you are connected to VPN and run the remote environment `(Ctrl+Shift+P) => Remote Containers: Open Folder in Container...`

Reviewer: What VPN do you need to connect to?

3. `yarn run start`

> 4. navigate to `http://localhost:9000/example`

Reviewer: ```suggestion
4. Navigate to <http://localhost:9000/example>
```

> ### Option 2:

Reviewer: Give this option a name/title as well.

> ## Local development

Reviewer: Since option 1 below is "Local," let's call this just "Development."

```suggestion
## Development
```

When a local console server is running, visit <http://localhost:9000/example>
to see the example plugin page.

### Option 2: Docker + VSCode Remote Container

> Make sure [Remote Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension is installed. 
> This method uses docker compose with one container being the console and the bridge,
> and the second container being the plugin with VSCode. After initial build, the cached containers will help you start developing in seconds.

Reviewer: ```suggestion
Make sure the [Remote Containers]
(https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
extension is installed. This method uses Docker Compose where one container is
the OpenShift console and the second container is the plugin. It requires that
you have access to an existing OpenShift cluster. After the initial build, the
cached containers will help you start developing in seconds.

```

> 1. Create a `dev.env` file inside `.devcontainer` folder with the correct values for your cluster:

Reviewer: ```suggestion
1. Create a `dev.env` file inside the `.devcontainer` folder with the correct values for your cluster:
```

```bash
OC_PLUGIN_NAME=my-plugin
OC_URL=https://api.example.com:6443
OC_USER=kubeadmin
OC_PASS=<password>
```

2. `(Ctrl+Shift+P) => Remote Containers: Open Folder in Container...`
3. `yarn run start`

> 4. Navigate to `http://localhost:9000/example`

Reviewer: ```suggestion
4. Navigate to <http://localhost:9000/example>
```

When a local console server is running, visit <http://localhost:9000/example>
to see the example plugin page.

### Option 2: Docker + VSCode Remote Container

> Make sure the [Remote Containers]
> (https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)

Reviewer: Sorry my previous suggestion broke the link rendering.

```suggestion
Make sure the
[Remote Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
```
